packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -115,19 +61,30 @@ contract Attestations is
   }
 
   // Stores attestations state for a single (identifier, account address) pair.
-  struct AttestationsMapping {
+  struct AttestedAddress {
+    // Number of requested attestations
+    uint32 requested;
     // Number of completed attestations
-    uint64 completed;
+    uint32 completed;
     // List of issuers responsible for attestations
     address[] issuers;
     // State of each attestation keyed by issuer
     mapping(address => Attestation) issuedAttestations;
   }
 
+  struct AttestationRequest {
+    // The block at which the attestations were requested
+    uint128 blockNumber;
+
+    // The number of attestations that were requested
+    uint128 attestationsRequested;
+  }
+
   struct IdentifierState {
     // All account addresses associated with this identifier
     address[] accounts;
-    mapping(address => AttestationsMapping) attestations;
+    mapping(address => AttestedAddress) attestations;
+    mapping(address => AttestationRequest) requests;
```

Comment:
Please add comments for these variables. In particular, it's non-intuitive that `requests` stores the most recent attestation request for this identifier/address pair for which issuers have not been selected, and is cleared out once issuers are selected. Once we have a term for this, we may want to update the variable name to capture this